packages/composites/glv-token/src/transport/utils.ts

```diff
@@ -0,0 +1,59 @@
+import Decimal from 'decimal.js'
+
+// The signed prices represent the price of one unit of the token using a value with 30 decimals of precision.
+export const SIGNED_PRICE_DECIMALS = 30
+
+export type PriceData = { [asset: string]: { bids: number[]; asks: number[] } }
+
+export type Source = { url: string; name: string }
+
+export const median = (values: number[]): number => {
+  if (values.length === 0) {
+    throw new Error('Input array is empty')
+  }
+
+  values = [...values].sort((a, b) => a - b)
+
+  const half = Math.floor(values.length / 2)
+
+  return values.length % 2 ? values[half] : (values[half - 1] + values[half]) / 2
+}
+
+/*
+Formats a given number with a specified precision without leading zeros and decimal point.
+Price decimals = SIGNED_PRICE_DECIMALS - token decimals
+
+toFixed(14.84329267, 18) -> '14843292670000'
+toFixed(0.99999558, 6) -> '999995580000000000000000'
+ */
+export const toFixed = (number: number, decimals: number): string => {
+  const n = new Decimal(number)
+  return n
+    .toFixed(SIGNED_PRICE_DECIMALS - decimals)
+    .replace('.', '')
+    .replace(/^0+/, '')
+}
+
+export interface Token {
+  symbol: string
+  address: string
+  decimals: number
+  synthetic: null | boolean
+}
+
+export interface Market {
+  marketToken: string
+  indexToken: string
+  longToken: string
+  shortToken: string
+  isListed: boolean
+}
+
+export const glvMarkets = {
+  '0x528A5bac7E746C9A509A1f4F6dF58A03d44279F9': 'ETH/USDC',
+  '0xdF03EEd325b82bC1d4Db8b49c30ecc9E05104b96': 'WBTC/USDC',
```

Comment:
Are these the only two supported input? We can put them in the the options of the input

Reply:
I really think we don't need this mapping. Only required input param should be GLV address. the contract returns both the long and short tokens